packages/pluggableWidgets/datagrid-web/src/components/ColumnSelector.tsx

```diff
@@ -17,21 +17,27 @@ export function ColumnSelector(props: ColumnSelectorProps): ReactElement {
     const optionsRef = useRef<HTMLUListElement>(null);
     const buttonRef = useRef<HTMLButtonElement>(null);
     const position = usePositionObserver(buttonRef.current, show);
+    const visibleCount = props.columns.length - props.hiddenColumns.length;
+    const isOnlyOneColumnVisible = visibleCount < 2;
```

Comment:
Is there a particular benefit to doing `<2` instead of `===1`?

Reply:
🤷‍♂️